Packs/OpenAI/Integrations/OpenAiChatGPTV3/OpenAiChatGPTV3.py

```diff
@@ -0,0 +1,153 @@
+            self._headers = {'Authorization': f"Bearer {self.api_key}", "Content-Type": "application/json"}
+            options = {"max_tokens": 1000, "model": "gpt-3.5-turbo", "messages": [{"role": "user", "content": prompt}]}
+        return requests.post(self.base_url, headers=self._headers, json=options)
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """
+    Tests API connectivity and authentication'
+    Returning 'ok' indicates that connection to the service is successful.
+    Raises exceptions if something goes wrong.
+    """
+    response = client.chatgpt('Hello!')
+    status = ''
+    try:
+        if response:
+            status = 'ok'
+            return status
+    except Exception as e:
+        exception_text = str(e).lower()
+        if 'forbidden' in exception_text or 'authorization' in exception_text:
+            status = 'Authorization Error: make sure API Key is correctly set'
+            return status
+        else:
+            raise e
+
+    return status
+
+
+def chatgpt_send_prompt_command(client: Client, prompt: str) -> CommandResults:
+    """
+    Command to send prompts to OpenAI ChatGPT API
+    and receive a response converted into json then
+    returned to Output function to convert it to markdown table
+
+    :type client: ``Client``
+    :param prompt:  arguments
+    """
+    if not prompt:
+        raise DemistoException('the prompt argument cannot be empty.')
+
+    response = client.chatgpt(prompt)
+
+    chatgpt_response = response.json()
+
+    return chatgpt_output(chatgpt_response)
+
+
+def chatgpt_output(response) -> CommandResults:
+    """
+    Convert response from ChatGPT to a human readable format in markdown table
+
+    :return: CommandResults return output of ChatGPT response
+    :rtype: ``CommandResults``
+    """
+    if response and isinstance(response, dict):
+
+        rep = json.dumps(response)
+        repJSON = json.loads(rep)
+        model = repJSON['model']
+        createdTime = repJSON['created']
+        id = repJSON['id']
+        choices = repJSON['choices'][0]['message']['content'].strip('\n')
+        promptTokens = repJSON['usage']['prompt_tokens']
+        completionTokens = repJSON['usage']['completion_tokens']
+        totalTokens = repJSON['usage']['total_tokens']
+        context = [{'ID': id, 'Model': model,
+                    'ChatGPT Response': choices, 'Created Time': createdTime,
+                    'Number of Prompt Tokens': promptTokens,
+                    'Number of Completion Tokens': completionTokens,
+                    'Number of Total Tokens': totalTokens
+                    }]
+
+    markdown = tableToMarkdown(
+        '### ChatGPT API Response ###',
+        context,
+        date_fields=['Created Time'],
+    )
+
+    results = CommandResults(
+        readable_output=markdown,
+        outputs_prefix='ChatGPTResponse',
+        outputs_key_field='id',
+        outputs=context
+    )
```

Comment:
Please move it inside the if statement, and handle the else condition if no result was returned.